spec/support/vcr_setup.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+ENV["RAILS_ENV"] ||= 'test'
+
+require 'vcr'
+
+VCR.configure do |config|
+  config.cassette_library_dir = "spec/fixtures/vcr_cassettes"
+  config.hook_into :webmock
+  config.default_cassette_options = { record: :once }
```

Comment:
This option is the default, I believe:
https://relishapp.com/vcr/vcr/v/6-1-0/docs/configuration/default-cassette-options